### DISCUSSION TITLE
[Request] For next build S1 F4

### DISCUSSION THREAD

**Discussion by fpmirabile** [mriscoc/Ender3V2S1#937] [Ideas]
I don't know if I'm the only one but, every time I update the firmware I'm getting issues with the CR-Touch mesh.

I found out, the issue is the probe amount setting, which is set to one. 
Everytime I set this to two or three, it fixes. So I'm wondering if there is any way you could change that setting for next builds.

If I'm the only one having this issue, I think we can ignore it :(

**Comment by mriscoc** [comment]
Use a [Configuration file](https://github.com/mriscoc/Ender3V2S1/wiki/Configuration-files) and set the number of probes with [`C851 M2`](https://github.com/mriscoc/Ender3V2S1/wiki/Custom-C-gcodes) for example.
